diff --git a/node_modules/fastify/lib/pluginUtils.js b/node_modules/fastify/lib/pluginUtils.js
index c390e95..27a1fa6 100644
--- a/node_modules/fastify/lib/pluginUtils.js
+++ b/node_modules/fastify/lib/pluginUtils.js
@@ -7,7 +7,6 @@ const {
   kTestInternals
 } = require('./symbols.js')
 const { exist, existReply, existRequest } = require('./decorate')
-const { FST_ERR_PLUGIN_VERSION_MISMATCH } = require('./errors')
 
 function getMeta (fn) {
   return fn[Symbol.for('plugin-meta')]
@@ -97,18 +96,6 @@ function _checkDecorators (that, instance, decorators, name) {
   })
 }
 
-function checkVersion (fn) {
-  const meta = getMeta(fn)
-  if (!meta) return
-
-  const requiredVersion = meta.fastify
-
-  const sanitizedVersion = this.version.replace(/-rc.\d+$/, '')
-  if (requiredVersion && !semver.satisfies(sanitizedVersion, requiredVersion)) {
-    throw new FST_ERR_PLUGIN_VERSION_MISMATCH(meta.name, requiredVersion, this.version)
-  }
-}
-
 function registerPluginName (fn) {
   const meta = getMeta(fn)
   if (!meta) return
@@ -120,7 +107,6 @@ function registerPluginName (fn) {
 
 function registerPlugin (fn) {
   registerPluginName.call(this, fn)
-  checkVersion.call(this, fn)
   checkDecorators.call(this, fn)
   checkDependencies.call(this, fn)
   return shouldSkipOverride(fn)
